src/navigation/context.py

```diff
@@ -15,6 +15,13 @@
 from std_msgs.msg import Time
 
 
+# read required parameters, if they don't exist an error will be thrown
+REF_LAT = rospy.get_param("gps_linearization/reference_point_latitude")
+REF_LON = rospy.get_param("gps_linearization/reference_point_longitude")
```

Comment:
Were these supposed to be used in convert()?

Reply:
Yea I think the same variables are declared again in the convert functions, do you mind just deleting the declarations in those functions?